core/client/db/src/lib.rs

```diff
@@ -531,8 +536,8 @@ impl<Block: BlockT<Hash=H256>> Backend<Block> {
 		Backend::from_kvdb(db as Arc<_>, config.pruning, canonicalization_delay)
 	}
 
-	#[cfg(test)]
-	fn new_test(keep_blocks: u32, canonicalization_delay: u64) -> Self {
+	#[cfg(any(test, feature = "test-helpers"))]
+	pub fn new_test(keep_blocks: u32, canonicalization_delay: u64) -> Self {
```

Comment:
As described in the issue, this should use `kvdb-inmemory`.

Reply:
It uses it, no? A couple of lines below:
```
let db = Arc::new(::kvdb_memorydb::create(NUM_COLUMNS));
```

Reply:
Ohh :( Sorry!